package.json

```diff
@@ -2,14 +2,23 @@
   "name": "sk-auth",
   "version": "0.3.7",
   "description": "Authentication library for use with SvelteKit featuring built-in OAuth providers and zero restriction customization!",
-  "main": "dist/index.js",
-  "module": "dist/index.esm.js",
-  "types": "dist/index.d.ts",
+  "module": "index.mjs",
```

Comment:
I'm sure I'm not alone when I say suggesting `.mjs` file extensions was one of the silliest proposals the Node maintainers have made. That would be like proposing `.njs` for anything which required a Node runtime. Only the latter makes more sense. Note with #78 the Node runtime is no longer needed for this library to function.

Reply:
I personally do not like this addition either, which is why I'll be closing it up now. Thanks for the review @vhscom!